src/tabs/tab.jsx

```diff
@@ -110,20 +111,12 @@ const Tab = React.createClass({
     const textColor = selected ? this.state.muiTheme.tabs.selectedTextColor : this.state.muiTheme.tabs.textColor;
 
     const styles = this.mergeStyles({
-      display: 'table-cell',
-      cursor: 'pointer',
-      textAlign: 'center',
-      verticalAlign: 'middle',
       paddingTop: (icon && !label) ? 4 : 0,
```

Comment:
Updated padding to have left and right padding, and set top and bottom padding to 0. It doesn't seem the conditional top padding is necessary with enhanced buttons. Let me know if you feel that it is necessary and I will put it back in.